cmd/admin-handlers.go

```diff
@@ -732,9 +734,12 @@ func writeSetConfigResponse(w http.ResponseWriter, peers adminPeers, errs []erro
 	// Build nodeResults based on error values received during
 	// set-config operation.
 	for i := range errs {
+		errSet := errs[i] != nil
+		errMsg := fmt.Sprintf("%v", errs[i])
 		nodeResults = append(nodeResults, nodeSummary{
-			Name: peers[i].addr,
-			Err:  fmt.Sprintf("%v", errs[i]),
+			Name:   peers[i].addr,
+			ErrSet: errSet,
+			ErrMsg: errMsg,
```

Comment:
For the sake of brevity and readability (less context/local variables to remember), I would rather write the previous two lines as:

```go
   ErrSet: errs[i] != nil,
   ErrMsg: fmt.Sprintf("%v", errs[i]),
```

and remove the two earlier assignments (lines 737 and 738).